sources/packages/api/src/database/entities/student.model.ts

```diff
@@ -77,4 +77,9 @@ export class Student extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   sinValidation: SINValidation;
+
```

Comment:
Please add comments

Reply:
Please match the comments with the DB also 😉